Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.schema.mof

```diff
@@ -1,7 +1,5 @@
-[ClassVersion("1.2.0")]
+[ClassVersion("1.0.0")]
```

Comment:
@D1v38om83r Worst case maybe you could make this 1.4.0.0 to match the resource version it will be shipped with? :)